agent/agents/process/process_logger.go

```diff
@@ -116,6 +117,27 @@ func replacer(redactWords []string) *strings.Replacer {
 	return strings.NewReplacer(r...)
 }
 
+var matchLogLevelRegex = regexp.MustCompile(`level=(\w+)`)
+
+func matchLogLevel(line string) logrus.Level {
+	matches := matchLogLevelRegex.FindStringSubmatch(line)
+	if len(matches) < 2 {
+		return logrus.InfoLevel
+	}
+
+	result, err := logrus.ParseLevel(matches[1])
+	if err != nil {
+		// unreachable
+		return logrus.ErrorLevel
```

Comment:
@ritbl it's small chance that we will have this error, but i cover this case in tests by msgNumber

Reply:
I think it is risky to return error here, it can be line that has missing log level, and it will be logs as error

plus, comment is misleading, it is totally possible to be reachable